MEP_Engine/Query/Geometry.cs

```diff
@@ -0,0 +1,43 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2020, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.Base;
+using BH.oM.Geometry;
+using BH.Engine.Geometry;
+using BH.oM.MEP.Elements;
+
+using System.ComponentModel;
+using BH.oM.Reflection.Attributes;
+using System.Collections.Generic;
+
+using System.Runtime.InteropServices;
+
+namespace BH.Engine.BuildingEnvironment
+{
+    public static partial class Query
+    {
+        public static IGeometry Geometry(this IFlow obj)
+        {
+            return Engine.Geometry.Create.Line(obj.StartNode.Position, obj.EndNode.Position);
```

Comment:
I would rather return `new Line { Start = obj.StartNode.Position, End = obj.EndNode.Position }` to avoid the extra call. But this is my preference, would not be desperate about it.